grpc.log

```diff
@@ -0,0 +1,71 @@
+github.com/kubermatic/machine-controller google.golang.org/grpc@v1.27.1
```

Comment:
What is the purpose of this file? I've never seen it before.

Reply:
This is something I used to investigate a conflict on grpc, I committed this by mistake. Thx for spotting it ;-)